lms/views/error.py

```diff
@@ -48,3 +50,15 @@ def error(self):
         return {'message': _("Sorry, but something went wrong. "
                              "The issue has been reported and we'll try to "
                              "fix it.")}
+
+
+    @view_config(context=MissingLTILaunchParamError)
+    def missing_lti_launch_param_error(self):
+        self.request.response.status_int = 400
+        # If code raises a MissingLTILaunchParamError it means that
```

Comment:
These comments are  really good. As they apply to the entire view handler method, how about converting them to a docstring?